### PR TITLE
Make blank cells in heatmap chart accessible

### PR DESCRIPTION
**Pull request by krkshitij**
## Previous Behavior

Blank cells in HeatMapChart are not accessible.

## New Behavior

After

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-charting/src/components/HeatMapChart/HeatMapChart.base.tsx [hunk 1]
```diff
@@ -294,45 +298,81 @@ export class HeatMapChartBase extends React.Component<IHeatMapChartProps, IHeatM
...
-              width={this._xAxisScale.bandwidth()}
-              height={this._yAxisScale.bandwidth()}
-              onClick={dataPointObject.onClick}
-            />
-            <text
-              dominantBaseline={'middle'}
-              textAnchor={'middle'}
-              className={this._classNames.text}
-              transform={`translate(${this._xAxisScale.bandwidth() / 2}, ${this._yAxisScale.bandwidth() / 2})`}
+    yAxisDataPoints.forEach((yAxisDataPoint: string) => {
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/HeatMapChart/HeatMapChart.base.tsx:301]: the snapshots should get updated with this change

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/HeatMapChart/HeatMapChart.base.tsx (hunk 1 above)
Comment (krkshitij) [reply packages/react-charting/src/components/HeatMapChart/HeatMapChart.base.tsx:301]: Most of the snapshot tests are behind the TEST env flag. But I see 2 RTL snapshot tests enabled, and I have no idea why these snapshots are not updating.

**Pull request merged**